proposals/mdawson-cpc-charter/CPC-Charter.md

```diff
@@ -44,125 +44,73 @@ direction of the policies set by the Board.
-
-  * If there are fewer Qualified Nominees than open TOC seats available for election by either the Governing Board, End User TAB or TOC, the group shall initiate another round of nominations with each member or individual eligible to nominate submitting at most one (1) nominee.
-
-* Constraints
-
-  *  TOC Members shall serve two-year, staggered terms. The initial six elected TOC members from the Governing Board election shall serve an initial term of three (3) years. The TOC members initially elected by the End User TAB and TOC shall serve an initial term of two (2) years.
-
- * TOC members may be removed by a two-thirds vote of the other TOC members, with the impacted individual ineligible to participate in the vote.
-
- * Any TOC member that misses three (3) consecutive meetings shall be automatically suspended from eligibility to vote until having attended two meetings consecutively. For avoidance of doubt, the suspended TOC member shall be eligible to vote in the second consecutive meeting.
-
- * The TOC charter, model, approach, composition, etc may be modified by a two-thirds vote of the entire Governing Board.
-
-===========
-
+The CPC is in inclusive group with 2 types of members:
+
+* Regular members
+* Voting members
+
+**Regular members** Regular members have the roles and responsibilities
+as outlined below. In any case where it is not specified that a role
+or responsibility applies to a Voting member, then a Regular member is
+implied. Anyone who has been a member of one of the projects under the 
+Foundation for at least 3 months may request to
+become a regular member by opening a PR to add themselves to the list of
+regular members. Regular members remain for as long as they are active
+within the work of the CPC. Regular members who have not been active
+in GitHub, participated in meetings, or other work of the CPC for 3 months may be
+removed from the list of Regular members. In addition, a Regular CPC member
+can be removed by voluntary resignation, or by a standard CPC motion.
+
+**Voting members** Voting members have the roles and responsibilities
+as outlined below. Voting members are selected as follows:
+
+* Each top level project may nominate 2 member through a process
+of their choosing. Once nominated the member must be ratified by the
+board before becoming a Voting member.
+
+* 2 Voting members may be nominated by the non top-level projects.
+If a vote is required among candidates nominated by different projects,
+each of the non-top level projects will have 2 votes (one for each position).
+
+* 2 Voting members may be nominated by the Regular members. Once
+nominated these members must be ratified by the board before
+becoming a Voting member.
+
+Voting members are expected to make a time commitment that
```

Comment:
If a member does not meet the expectations then it would be up to the rest of the CPC to ask that they step down as per 
>Voting members serve for a term of 1 year and must be re-nominated
and ratified by the board each year. In addition, a Voting CPC member
can be removed by by voluntary resignation, by a standard CPC motion,
or in accordance to the participation rules described for Regular
members.

In terms of participating in "all voting matters" I add the clarification that means vote or abstain.  I'd like to leave that as the expectation with the understanding that in reality, all members will not necessarily be able to vote (holidays etc.) but that they will make all efforts to do so.